Makefile

```diff
@@ -1,17 +1,20 @@
+GOFILES=$(shell find . -name \*.go)
+PODFILES=$(shell find . -name \*.go -o -name Dockerfile -o -name \*.yaml -o -name \*.yaml.in -o -name \*.tpl)
```

Comment:
I really don't like that. All manifests should be in the manifest directory. Further we hack enough around already in the "hack" folder, no need to hack here again.